src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeHttpWrapperTests.cs

```diff
@@ -70,11 +73,42 @@ public async Task PostAsync_ExecutesRequestsOnTheCorrectPort()
             var startUp = SetupServerStartUp(port: 1234);
             var testSubject = CreateTestSubject(startUp.Object, httpMessageHandler);
 
-            await testSubject.PostAsync("some-url");
+            await testSubject.PostAsync("some-url", null, CancellationToken.None);
 
             requestUri.Should().BeEquivalentTo(new Uri("http://localhost:1234/some-url"));
         }
 
+        [TestMethod]
+        public async Task PostAsync_PassesCancellationToken()
+        {
+            var validationPassed = false;
+            var httpMessageHandler = new Mock<HttpMessageHandler>();
+            var originalTokenSource = new CancellationTokenSource();
+
+            httpMessageHandler
+                .Protected()
+                .Setup<Task<HttpResponseMessage>>("SendAsync",
+                    ItExpr.IsAny<HttpRequestMessage>(),
+                    ItExpr.IsAny<CancellationToken>())
+                .ReturnsAsync(new HttpResponseMessage {Content = new StringContent("response")})
+                .Callback((HttpRequestMessage message, CancellationToken receivedToken) =>
+                {
+                    // SendAsync uses `CancellationTokenSource.CreateLinkedTokenSource`
```

Comment:
I couldn't do a simple comparison of receivedToken and the original token, as they're actually not the same token -- decompiled HttpClient shows that it creates CancellationTokenSource.CreateLinkedTokenSource under the covers, so I had to get creative.